Header header
float64 rawhid_recv  # [seconds]
float64 ros_spin_once  # [seconds]
float64 check_checksum  # [seconds]
float64 parse_header  # [seconds]
float64 parse_imu  # [seconds]
float64 parse_strobe  # [seconds]
float64 compute_strobe_total # [seconds]
float64 publish_imu_raw  # [seconds]
float64 publish_strobe_raw  # [seconds]
float64 push_imu  # [seconds]
float64 push_strobe  # [seconds]
float64 compute_offsets  # [seconds]
float64 filter_imu  # [seconds]
float64 publish_imu  # [seconds]
float64 associate  # [seconds]
float64 publish_camera  # [seconds]
float64 update  # [seconds]
float64 period  # [seconds]